packages/prisma/schema.prisma

```diff
@@ -1379,3 +1380,15 @@ model PlatformBilling {
 
   team Team @relation(fields: [id], references: [id], onDelete: Cascade)
 }
+
+model AuditLog {
+  id         Int      @id @default(autoincrement())
+  action     String
+  actionDesc String?
+  actor      Json
```

Comment:
Instead of Json can we turn this into `actorId` so we can query/join the user on query? We're trying to avoid Json fields as much as possible are these are harder to index and query.

Reply:
Done.